### PR TITLE
Removes {} from console logging

### LINKED ISSUE #1549: Console logging shows `{}` values where variables or nothing should appear
**Describe the bug**

The console logging shows `{}` values where variables or nothing should appear

**To Reproduce**

1. Start the Admin app
2. The `{}` entries are visible in the console logs

**Expected behavior**
- The  `{}` values should be populated by variables or removed if adding variables has no use case

**Screenshots**
image

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by RAGHAV-N5**
**What kind of change does this PR introduce?**

Bugfix

**Issue Number:**

Fixes #1549 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

Screenshot 2023-12-31 101654

**If relevant, did you update the documentation?**

No

**Summary**

This PR aims to remove {} from the console logging which was not necessary. 

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Review by noman2002 — CHANGES_REQUESTED** [review]
Fix the failing test.

**Comment by RAGHAV-N5** [comment]
@noman2002 Can you please help me how to proceed to fix the failing tests?

**Comment by palisadoes** [comment]
Please read the error messages the answer should be there.

**Comment by RAGHAV-N5** [comment]
@palisadoes 
Are these the ones I should fix?
Screenshot 2024-01-01 093839

**Comment by palisadoes** [comment]
Please click on the link to read the cause of the failure.

image

File: setup.ts [hunk 1]
```diff
@@ -45,113 +44,6 @@ async function accessAndRefreshTokens(
   }
 }
 
-// Check connection to Redis with the specified URL.
```
Comment (palisadoes) [inline setup.ts:48]: 1. There is no need to have this file included in this PR
1. You have also removed key functionality in it.
1. Exclude this file from the PR
1. Merge with the latest upstream also

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Comment by RAGHAV-N5** [comment]
@palisadoes
 I was using nodejs version of 21.4.0 but after carefully examining the workflow files I found that node js [20.x] was used. 
 So, this is causing the tests to fail. Please correct me if I'm wrong.

**Comment by palisadoes** [comment]
> @palisadoes I was using nodejs version of 21.4.0 but after carefully examining the workflow files I found that node js [20.x] was used. So, this is causing the tests to fail. Please correct me if I'm wrong.

Please use your initiative to see whether that is the cause.

**Pull request closed without merging**